viewpager/ViewPager.js

```diff
@@ -2,183 +2,229 @@
  * Created by tangzhibin on 16/2/28.
  */
 
-'use strict'
+"use strict";
 
-import { PanResponder, Platform, ScrollView, StyleSheet, View, ViewPagerAndroid } from 'react-native'
-import React, { Component } from 'react'
+import {
+  PanResponder,
+  Platform,
+  ScrollView,
+  StyleSheet,
+  View
+} from "react-native";
+import ViewPagerAndroid from "react-native-viewpager";
```

Comment:
isn't that supposed to be 
`import ViewPagerAndroid from "@react-native-community/viewpager";`
?

Reply:
Yup! Updated, thanks (although have a feeling the PR is somewhat optimistic given the last commit to master)